src/Battle.php

```diff
@@ -0,0 +1,211 @@
+<?php
+declare(strict_types=1);
+
+namespace LotGD\Core;
+
+use LotGD\Core\{
+    DiceBag,
+    Exceptions\ArgumentException,
+    Exceptions\BattleIsOverException,
+    Exceptions\BattleNotOverException,
+    Models\FighterInterface
+};
+
+/**
+ * Description of Battle
+ */
+class Battle
+{
+    const DAMAGEROUND_PLAYER = 0b01;
+    const DAMAGEROUND_MONSTER = 0b10;
+    const DAMAGEROUND_BOTH = 0b11;
+    
```

Comment:
Then they (or we) can introduce a new Battle Class doing the same. I'll implement a BattleInterface.

(I totally forgot than the new version has this multi-combatants-fight, 0.9.7 did not have this.)